lib/url-assembler-factory.js

```diff
@@ -83,6 +83,8 @@ module.exports = function (request) {
     if (typeof key === 'object') {
       return _multiParam(this, key, (value === true));
     }
+    key = encodeURI(key);
+    value = encodeURI(value);
```

Comment:
@azu That does not seem right. Feel free to make a pull request to safe-url-assembler and I'll release a new version.